src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -245,7 +245,8 @@ var ReactCompositeComponentMixin = {
     inst.props = publicProps;
     inst.context = publicContext;
     inst.refs = emptyObject;
-    inst.updater = ReactUpdateQueue;
+    inst.updater = transaction.getUpdateQueue ?
```

Comment:
Okay, that makes complete sense. I will consider it is required. Should I worry about other Transaction files though ? And yes, there are some other `ReactUpdateQueue` usages I didn't catch, I will update them as well!